lib/tbot/tbot.go

```diff
@@ -259,17 +260,23 @@ func (b *Bot) Run(ctx context.Context) error {
 		cancel()
 		return nil
 	})
-	if b.cfg.Debug && b.cfg.DiagAddr != "" {
+
+	if b.cfg.DiagAddr != "" {
 		eg.Go(func() error {
 			b.log.WithField("addr", b.cfg.DiagAddr).Info(
-				"DiagAddr configured, diagnostics service will be started.",
+				"diag_addr configured, diagnostics service will be started.",
 			)
 			mux := http.NewServeMux()
-			mux.HandleFunc("/debug/pprof/", pprof.Index)
-			mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
-			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
-			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
-			mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+			mux.Handle("/metrics", promhttp.Handler())
+			// Only expose pprof when `-d` is provided.
+			if b.cfg.Debug {
```

Comment:
This is technically a breaking change no? Are we worried about breaking users who are using the pprof endpoints without the `-d` flag today?

Reply:
The if statement before this block used to be `if b.cfg.Debug && b.cfg.DiagAddr != "" {` which prevented this server even being started if `-d` was not provided.